### PR TITLE
Add ROE functionality for address/domain registering

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

**Comment by jcalfee** [comment]
Are you setting the price in the back-end during the purchase?  The UI is a display price but it does not get to decide the real price.

File: server/routes/wallet.js [hunk 1]
```diff
@@ -281,7 +298,18 @@ router.post('/public-api/buy-address', handler(async (req, res) => {
   }
 
   const {name, logo_url} = wallet
-  const price = +Number(wallet[`${type}_sale_price`])
+  let price = +Number(wallet[`${type}_sale_price`])
+  
+  if (wallet[`${type}_roe_active`]) {
```
Comment (andreyvEze) [inline server/routes/wallet.js:303]: @jcalfee yes, here

File: server/roe.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+const JsonFetch = require('../src/api/json-fetch');
+const db = require('./db/models');
+
+const binanceEndpoint = 'https://api.binance.com/api/v3/'
```
Comment (ericbutz) [inline server/roe.js:4]: @pawmmm Is there a limit on how often we can hit Binance for rates?

Comment (pawmmm) [reply server/roe.js:4]: 1200 requests per minute

**Pull request merged**